SOURCES/etc/xapi.d/plugins/hyperthreading.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import errno
+import json
+import subprocess
+import sys
+
+import XenAPIPlugin
+
+sys.path.append('.')
+from xcpngutils import configure_logging, run_command
+
+
+# returns a JSON dict {<poolname>: {mountpoint: <mountpoint>, ...}}
+# xe host-call-plugin host-uuid=<UUID> plugin=zfs.py fn=list_zfs_pools
```

Comment:
The comment is incorrect.

Reply:
@nraynaud can you address it?

Reply:
sorry, I pushed a fix yesterday, but I didn't know the branch has already been deleted, I need to send a new PR.

Reply:
This function returns always `"false"` as a string ! 
do all `vates lab` hosts not support `hyperthreading`  ?
Can you be more specific of what this function returns please :slightly_smiling_face:

Reply:
Dell T30 we have in our lab are Xeon E3-1225 v5. Those CPUs aren't equipped with HT, as [Intel documentation explains](https://ark.intel.com/content/www/fr/fr/ark/products/88168/intel-xeon-processor-e3-1225-v5-8m-cache-3-30-ghz.html).

However, you can try on `192.168.100.3` (same lab credentials), which is a Ryzen CPU with SMT (equivalent of HT but for AMD).

Reply:
@olivierlambert thank you

Reply:
According to my tests, this function returns : 
- `false` when there's no hyperthreading
- `true` when there's hyperthreading
Which is completely expected.
Maybe its name should be `is_hyperthreading`  instead of `get_hyperthreading`